python/cudf/cudf/core/dataframe.py

```diff
@@ -5891,6 +5891,41 @@ def _from_columns(cls, cols, index=None, columns=None):
 
         return cls(data=data, index=index,)
 
+    @copy_docstring(Frame._interpolate)
```

Comment:
If you're not modifying the docstring at all it should be inherited even if you override the function.
```
In [84]: class A:
    ...:     def f(self):
    ...:         """My docstring."""
    ...:         pass
    ...:

In [85]: class B(A):
    ...:     def f(self):
    ...:         pass
    ...:

In [86]: help(B().f)
Help on method f in module __main__:

f() method of __main__.B instance
    My docstring.
```